### PR TITLE
Add openregister configs

### PR DESCRIPTION
**Pull request by chr1sdi**

### CODE REVIEW

File: ansible/upload_configs_to_s3.yml [hunk 1]
```diff
@@ -10,9 +10,11 @@
     services:
       mint: 'mint-config.yaml'
       presentation: 'config.yaml'
+      openregister: 'openregister-config.yaml'
```
Comment (philandstuff) [inline ansible/upload_configs_to_s3.yml]: this should just be `config.yaml`

File: ansible/create_databases.yml [hunk 2]
```diff
@@ -24,3 +24,11 @@
         write_user: 'mint',
         master_password: "{{ lookup('pass', '{{ vpc }}/rds/mint/master') }}"
       }
+
+    - { role: database,
+        inventory_group_name: '{{ vpc }}-openregister-db',
+        read_user: 'openregister',
```
Comment (philandstuff) [inline ansible/create_databases.yml]: this needs more thought

Comment (chr1sdi) [inline ansible/create_databases.yml]: it should only add permission to an existing user:
http://docs.ansible.com/ansible/postgresql_user_module.html#synopsis

the order of creating users is WRITE, READ so the second call (privilege `SELECT`) shouldn't do anything as WRITE has `ALL`

File: ansible/roles/service_config/templates/openregister-config.yaml.j2 [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  initialSize: {{ db_initial_size | default(1) }}
+  minSize: {{ db_min_size | default(1) }}
+  maxSize: {{ db_max_size | default(4) }}
+
+  properties:
+    charSet: UTF-8
+
+registerDomain: {{ register_domain }}
+
+enableDownloadResource: {{ settings.enable_download_resource | default(true) }}
```
Comment (philandstuff) [inline ansible/roles/service_config/templates/openregister-config.yaml.j2]: this isn't a complete, valid config file for openregister-java. In particular, it's missing the `register:` field.

**Pull request merged**